external/mmat.m

```diff
@@ -51,9 +52,46 @@
 nDB = ndims(B);
 nD = max(nDA,nDB);
 
+if nD == 2 && isdefaultdim
+    C = A * B;
```

Comment:
Good to have this shortcut I think, perhaps reduces the overhead of calling `mmat` for 2D arrays - even though we tend to avoid it!